openshift/templates/aries-endorser-agent/aries-endorser-agent-deploy.webvh-sandbox.param

```diff
@@ -26,13 +27,15 @@ ACAPY_MONITOR_REVOCATION_NOTIFICATION=true
 AGENT_ADMIN_PORT=8051
 AGENT_HTTP_PORT=8050
 AGENT_WS_PORT=8052
-AGENT_BASE_URL=https://sovrin-endorser-agent-test.apps.silver.devops.gov.bc.ca
-AGENT_BASE_WS_URL=wss://sovrin-endorser-agent-test.apps.silver.devops.gov.bc.ca
+AGENT_BASE_URL=https://aries-endorser-agent-webvh-sandbox.apps.silver.devops.gov.bc.ca
+AGENT_BASE_WS_URL=wss://aries-endorser-agent-webvh-sandbox.apps.silver.devops.gov.bc.ca
 ACAPY_IMAGE_URL=
-ACAPY_GENESIS_URL=https://raw.githubusercontent.com/sovrin-foundation/sovrin/stable/sovrin/pool_transactions_sandbox_genesis
+ACAPY_GENESIS_URL=
+ACAPY_PLUGIN=[webvh]
+ACAPY_PLUGIN_CONFIG=<--set config in here once configmap created-->
```

Comment:
This is the default value being set in the configmap, it will need to be manually set on creation and after every update as of now. It is an interim quick-fix until we publish a chart for the endorser service that would not require manual intervention.